### PR TITLE
fix an error with Vietnamese which can't use in command line

### PR DESCRIPTION
**Pull request by dongchirua**
### Changes proposed in this pull request:

* Add support for Vietnamese in the command line

### Status

- [x ] READY
- [] HOLD
- [] WIP (Work-In-Progress)

### How to verify this change
At the command line, run `num2words {a_number_you want} -l vi ` or at code you can run `num2words("{a_number_you_want}", lang="vi")`. For example, `num2words("21", lang="vi")` should return "hai mươi mốt"

### CODE REVIEW

**Comment by dongchirua** [comment]
[Build Status](https://travis-ci.com/dongchirua/num2words)

**Comment by dongchirua** [comment]
is there anyone here who can review and approve?

**Comment by mromdhane** [comment]
@dongchirua Yes, I will review it, once the CI check is complete.

File: tests/test_vi.py [hunk 1]
```diff
@@ -137,3 +137,6 @@ def test_special_number(self):
             num2words(1000101017, lang="vi"),
             "một tỷ một trăm lẻ một nghìn lẻ mười bảy"
         )
+
+    def test_str_to_number(self):
+        self.assertEqual(num2words("21", lang="vi"), "hai mươi mốt")
```
Comment (erozqba) [inline tests/test_vi.py:142]: @dongchirua this library is intended to transform `numbers` not `strings` into words -> strings

**Review by erozqba — COMMENTED** [review]
We don't intend to support strings as numbers.

**Pull request closed without merging**